src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -433,6 +433,11 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
 
       isClosingEmailEnabled: this.sessionEditFormModel.isClosingEmailEnabled,
       isPublishedEmailEnabled: this.sessionEditFormModel.isPublishedEmailEnabled,
+
+      studentDeadlines: {},
+      instructorDeadlines: {},
+
+      isGoingToNotifyAboutDeadlines: false,
```

Comment:
I've decided to just plug it back as-is, as they'll be retrieved either way under the session

Reply:
Actually I realized it may actually be better to make the fields optional. I'll liaise with Jay about it.

Reply:
> Actually I realized it may actually be better to make the fields optional. I'll liaise with Jay about it.

As discussed on slack earlier, ideally we want to try to keep this here because it is a` POST` and not a `PATCH` request. Will also simplify updating logic.

Reply:
Got it, thanks!